assets/js/googlesitekit/api/index.js

```diff
@@ -1,5 +1,79 @@
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
```

Comment:
I've added this so we can better raise errors in our APIs going forward, especially for public ones. It's quite a nice way of doing error handling (especially because we aren't using types) for functions like `get()` and `set()` that have a lot of positional arguments easily omitted.